packages/components/src/components/SegmentedControl/styles/segmentedControlStyles.ts

```diff
@@ -9,7 +11,8 @@ export const segmentedControlItemStyles =
     // Apply border radius based on the index
     'first:tw-rounded-l-[inherit] [&:nth-last-child(2)]:tw-rounded-r-[inherit] ' +
     // Focus styles for keyboard navigation
-    'focus-visible:tw-rounded-[inherit] focus-visible:tw-outline focus-visible:tw-outline-blue';
+    'focus-visible:tw-rounded-[inherit] focus-visible:tw-outline focus-visible:tw-outline-blue' +
```

Comment:
The component already has `focus-visible:tw-outline focus-visible:tw-outline-blue` make sure to clean it up